HCore-Storage/Client/Impl/GoogleCloudStorageClientImpl.cs

```diff
@@ -299,7 +299,7 @@ public async Task<string> GetSignedDownloadUrlAsync(string containerName, string
                 .CreateScoped(new string[] { "https://www.googleapis.com/auth/devstorage.read_only" })
                 .UnderlyingCredential as ServiceAccountCredential;
 
-            var urlSigner = UrlSigner.FromServiceAccountCredential(credential);
+            var urlSigner = UrlSigner.FromCredential(credential);
```

Comment:
warning CS0618: 'UrlSigner.FromServiceAccountCredential(ServiceAccountCredential)' is obsolete: 'Use FromCredential(ServiceAccountCredential) which is equivalent.'

Reply:
Is this tested?

Reply:
I still remember that there were issues with the signed link not working anymore...

Reply:
Ok, do you want me to skip the google packages update for now?

Reply:
Now it works with my changes